### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by kwsorensen**
#7

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+@kwsorensen or @aro5000.
```
Comment (aro5000) [inline CODE_OF_CONDUCT.md:63]: Should we link our GitHub pages here @kwsorensen? Or does that happen automatically in a README?

**Review by aro5000 — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (kwsorensen) [reply CODE_OF_CONDUCT.md:63]: Not sure. we can find out and fix it if it doesn't.

Comment (kwsorensen) [reply CODE_OF_CONDUCT.md]: They didn't. added our github pages.

Comment (aro5000) [reply CODE_OF_CONDUCT.md]: LGTM!

**Pull request merged**